### PR TITLE
fix(client_openxr): :bug: Fix head angular velocity

### PR DESCRIPTION
**Pull request by zmerp**
This *might* fix the prediction failure reported by JXI

### CODE REVIEW

**Review by The-personified-devil — COMMENTED** [review]
let's see if it works I guess

File: alvr/session/src/settings.rs [hunk 1]
```diff
@@ -1324,6 +1324,11 @@ pub struct ExtraConfig {
     pub capture: CaptureConfig,
     pub logging: LoggingConfig,
     pub patches: Patches,
+
+    #[cfg_attr(not(debug_assertions), schema(flag = "hidden"))]
+    #[schema(strings(help = "Multiply linear and angular velocity by 10 for debug purposes."))]
+    pub magnify_velocities: bool,
```
Comment (The-personified-devil) [inline alvr/session/src/settings.rs:1330]: might as well make it a slider atp

Comment (zmerp) [reply alvr/session/src/settings.rs:1330]: well yeah

File: alvr/client_core/src/connection.rs [hunk 2]
```diff
@@ -203,6 +204,8 @@ fn connection_pipeline(
     let settings = stream_config.settings;
     let negotiated_config = stream_config.negotiated_config;
 
+    *ctx.velocities_multiplier.write() = settings.extra.velocities_multiplier;
```
Comment (The-personified-devil) [inline alvr/client_core/src/connection.rs:207]: This doesn't actually update dynamically or am I missing something?

Comment (zmerp) [reply alvr/client_core/src/connection.rs:207]: nope, it doesn't update automatically. also it's not a slider. I don't think we should put any more effort into this. this is just for debug

Comment (The-personified-devil) [reply alvr/client_core/src/connection.rs:207]: Fair, I just don't want it to confusing when someone's modifying it for debugging purposes and they don't know that it doesn't update dynamically. So probably put a comment onto the setting

Comment (zmerp) [reply alvr/client_core/src/connection.rs:207]: done, but i think that was unnecessary because the user can see it doesn't have the green dot

Comment (The-personified-devil) [reply alvr/client_core/src/connection.rs:207]: But it's also not marked as `steamv-restart`, so yknow

**Review by The-personified-devil — APPROVED** [review]

**Pull request merged**